### ISSUE TITLE
Cannot install `API` when using Node.js v19

### ISSUE THREAD

**Issue by jhreis** [readmeio/api#590]
Running Node v19 (current release) and appears `API` is restricted to `<18` as for node version. 

Looks like this can be ignored via [`yarn install --ignore-engines`](https://stackoverflow.com/questions/56617209/the-engine-node-is-incompatible-with-this-module), but would prefer not to suppress this.

(screenshot copy)
>➜ yarn add api
yarn add v1.22.19
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error postman-to-openapi@2.9.0: The engine "node" is incompatible with this module. Expected version ">=12 <18". Got "19.1.0"
error Found incompatible module.

![➜ yarn add api
yarn add v1.22.19
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error postman-to-openapi@2.9.0: The engine "node" is incompatible with this module. Expected version ">=12 <18". Got "19.1.0"
error Found incompatible module.](https://user-images.githubusercontent.com/2781690/206761889-ebb47446-b90b-4f96-84fa-78ea6a6b739e.png)

**Comment by erunion** [comment]
I've got a fix for this in `postman-to-openapi` [here](https://github.com/joolfe/postman-to-openapi/pull/247), when it's incorporated I'll pull it in here.

**Comment by thescientist13** [comment]
FWIW, this also occurs for Node 18 as well (if it helps for updating the title)
```sh
% yarn add api
yarn add v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error postman-to-openapi@2.9.0: The engine "node" is incompatible with this module. Expected version ">=12 <18". Got "18.12.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

But looks like the above mentioned PR will account for this as well.  🙌

**Comment by erunion** [comment]
This is now resolved with v5.0.6.
